### DISCUSSION TITLE
fCal spatial calibration using customized N-wire pattern

### DISCUSSION THREAD

**Discussion by yichengggg** [PlusToolkit/PlusLib#1202] [Q&A]
Hello, 

I am trying to calibrate our curvilinear probe using the fCal 3.0 phantom. 
We wired the phantom following your N-wire suggestion, and we have a pattern of 2 "N"s on a same level (shows below) but seems like fCal cannot correctly recognize the lines (which are represented with 6 points) giving issues with spatial calibration. 
We are wondering if this wiring pattern is recommended within the phantom? Or should we specify the `Type` parameter in the configuration file under `<PhantomDefinition>`? We see options of "Triple-N" and "Multiple-N" but now sure if there is anything to do with this. 

Thank you!

![image](https://github.com/PlusToolkit/PlusLib/assets/62568712/c3d58b5c-bc94-4a70-bb62-0f05e691f15d)

**Comment by ungi** [comment]
Hi, I think only the point on the middle wire is used for calibration, so you would need at least three N's and not in the same level.
However, it's also important to note that N-wire calibration is very difficult to implement and only worth it if automatic calibration is necessary by clinician users at the bedside regularly. Otherwise, you can save a lot of time and get better accuracy by using stylus-based calibration.
